types/common.d.ts

```diff
@@ -7,6 +7,7 @@ interface IConfig {
   autoBreadcrumbs?: boolean;
   autoNotify?: boolean;
   appVersion?: string;
+  autoCaptureSessions?: boolean;
   endpoint?: string;
   sessionEndpoint?: string;
   autoCaptureSessions?: boolean;
```

Comment:
It's already in the definition for this type?

Reply:
argh!!!!!  apologies, Ben.  :)